### PR TITLE
[hotfix][doc] Fix typo in analyze.md

### PR DESCRIPTION
**Pull request by jasonyuan-cn**
Fixed the SQL sample syntax error in analyze.md, the comma in the sample SQL should be inside the double quotes

### CODE REVIEW

File: docs/content/docs/dev/table/sql/analyze.md [hunk 1]
```diff
@@ -379,4 +379,4 @@ ANALYZE TABLE [catalog_name.][db_name.]table_name PARTITION(partcol1[=val1] [, p
 
 *NOTE:* For the fix length types (like `BOOLEAN`, `INTEGER`, `DOUBLE` etc.), we need not collect the `avgLen` and `maxLen` from the original records.
 
-{{< top >}}
\ No newline at end of file
+{{< top >}}
```
Comment (reswqa) [inline docs/content/docs/dev/table/sql/analyze.md:382]: What is the purpose of this change?

File: docs/content/docs/dev/table/sql/analyze.md [hunk 2]
```diff
@@ -81,8 +81,8 @@ tableEnv.executeSql(
         " `id` BIGINT NOT NULl," +
         " `product` VARCHAR(32)," +
         " `amount` INT," +
-        " `sold_year` BIGINT", +
-        " `sold_month` BIGINT", +
+        " `sold_year` BIGINT," +
+        " `sold_month` BIGINT," +
```
Comment (reswqa) [inline docs/content/docs/dev/table/sql/analyze.md:85]: Please also check the Chinese document to see if there are any similar issues.

**Review by reswqa — COMMENTED** [review]
Thanks @jasonyuan-cn for the fix, I only left some minor comments, PTAL.

**Comment by reswqa** [comment]
BTW, you should rename your commit titile to `[hotfix][doc] Fix typo in analyze.md`.

File: docs/content/docs/dev/table/sql/analyze.md (hunk 1 above)
Comment (jasonyuan-cn) [reply docs/content/docs/dev/table/sql/analyze.md:382]: This line of code should not be changed, sorry for my carelessness. I will undo this line.

File: docs/content/docs/dev/table/sql/analyze.md (hunk 2 above)
Comment (jasonyuan-cn) [reply docs/content/docs/dev/table/sql/analyze.md:85]: Thanks for your reply. I will check Chinese document also.

**Comment by reswqa** [comment]
> BTW, you should rename your commit titile to [hotfix][doc] Fix typo in analyze.md.

Sorry, I may not have expressed myself clearly. What needs to be modified is not only the title of this pull request, but also the title of the commit message. Now it is: [fix: analyze.md SQL syntax error](https://github.com/apache/flink/pull/22615/commits/5e50d9b947ddb87cd8e7e700fce8c8af750ec2b9)

**Comment by jasonyuan-cn** [comment]
> > BTW, you should rename your commit titile to [hotfix][doc] Fix typo in analyze.md.
> 
> Sorry, I may not have expressed myself clearly. What needs to be modified is not only the title of this pull request, but also the title of the commit message. Now it is: [fix: analyze.md SQL syntax error](https://github.com/apache/flink/pull/22615/commits/5e50d9b947ddb87cd8e7e700fce8c8af750ec2b9).

Thank you so much for your patience and help. I have learned that. And I am trying to push new commit on this PR

**Comment by jasonyuan-cn** [comment]
I'm sorry, I'm too unfamiliar with github PRs, I'll close this one just in case, then I'll try to resubmit the PR to fix the typo problem.
